src/helpers/embedHelper.ts

```diff
@@ -1,4 +1,14 @@
+import type { FourNumberTuple } from "../types";
+
 export const getEmbedCode = (url: URL, embedParams: EmbedParams) => {
+  //
+  // TODO: don't mutate a function parameter
+  //
+  // delete bounds property if not embedding in viewport mode
+  if (embedParams.embedView !== "viewport") {
+    delete embedParams.embedBounds;
```

Comment:
TODO

Reply:
@petem-methods I think this is sorted now - removed it from the function call instead, when its not needed